eth/downloader/peer.go

```diff
@@ -386,9 +379,8 @@ func (ps *peerSet) Register(p *peer) error {
 
 	// Register the new peer with some meaningful defaults
 	ps.lock.Lock()
-	defer ps.lock.Unlock()
```

Comment:
What's the rationale behind this change? Code before change is pretty equivalent?

Reply:
IDK - came with the merge. Hadn't looked closely. I prefer the `defer` though ;)